discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -65,7 +65,8 @@ def _terraform_params(self, **kwargs):
                   # TODO change to namespace index
                   "libvirt_secondary_network_if": self.network_conf.libvirt_secondary_network_if,
                   "provisioning_cidr": self.network_conf.provisioning_cidr,
-                  "running": True
+                  "running": True,
+                  "single_node_ip": kwargs.get('single_node_ip', ''),
```

Comment:
The terrraform get this param?

Reply:
yes
so far it's got changed only on runtime, that's why I needed to add it here